src/cljs/main/broadfcui/common/style.cljs

```diff
@@ -62,11 +62,19 @@
 (defn create-form-label [text]
   [:div {:style {:marginBottom "0.16667em" :fontSize "88%"}} text])
 
+;; jQuery3+React warns about nil :value elements. Replace with empty string.
+(defn- replace-nil-value-key [props]
+  (reduce-kv (fn [m k v]
+               (if (and (= k :value) (nil? v))
+                 (assoc m k "")
+                 (assoc m k v)))
+             {} props))
```

Comment:
Gotcha ... understood! I'm okay with the empty string ... I believe this is used only for `<input type="text"`, right?